doc/tutorials/setup_logical_partition_scenario_on_nuc7i7dnb.rst

```diff
@@ -0,0 +1,264 @@
+   
+      If you update kernel image (bzImage) and modules on ubuntu of NUC7i7DNB,
+      need use ``scp`` to copy it from the remote host building machine.
+
+Update ACRN hypervisor Image
+****************************
+
+#. Before building the ACRN hypervisor, you need to figure out the I/O
+   address of the serial port, and the PCI BDF addresses of the SATA controller
+   and the USB controllers on NUC7i7DNB board. Enter the following command to
+   get the I/O addresses of the serial port. NUC7i7DNB boards support one
+   serial port ttyS0. Need to connect the serial port to the development host,
+   in order to access the ACRN serial console to switch between pre-launched
+   VMs:
+   
+   .. code-block:: none
+
+      $ dmesg | grep ttyS0
+      [    0.000000] console [ttyS0] enabled
+      [    1.562546] 00:01: ttyS0 at I/O 0x3f8 (irq = 4, base_baud = 115200) is
+      a 16550A
+
+   The following command prints detailed information about all PCI buses and
+   devices in the system:
+
+   .. code-block:: none
+
+      $ sudo lspci -vv
+      00:14.0 USB controller: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller (rev 21) (prog-if 30 [XHCI])
+              Subsystem: Intel Corporation Sunrise Point-LP USB 3.0 xHCI Controller
+      00:17.0 SATA controller: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode] (rev 21) (prog-if 01 [AHCI 1.0])
+              Subsystem: Intel Corporation Sunrise Point-LP SATA Controller [AHCI mode]
+      00:1f.6 Ethernet controller: Intel Corporation Ethernet Connection I219-LM (rev 21)
+              Subsystem: Intel Corporation Ethernet Connection I219-LM
+
+   .. note:: 
+
+      Double check PCI devices BDF defined in the ``hypervisor/arch/x86/configs/nuc7i7bnh/pci_devices.h``.
+
+#. Clone the ACRN source code and configure the build options
+
+   Please refer :ref:`getting-started-building` to setup ACRN build envniroment
+
+   Clone the ACRN source code:
+
+   .. code-block:: none
+
+      $ git clone https://github.com/projectacrn/acrn-hypervisor.git
+      $ cd acrn-hypervisor/hypervisor
+
+   Configure the build options:
+
+   .. code-block:: none
+   
+      $ make defconfig BOARD=nuc7i7bnh
+      $ make menuconfig
+
+   Updates the following configure item:
+
+     *  Set ACRN Scenario as "Logical Partition VMs";
+     *  Set Hypervisor mode as "Partition mode";
+     *  Set Target board as "nuc7i7bnh"
+     *  Set Maximum number of VCPUs per VM as "2";
+     *  Set Maximum number of PCPU as "4";
+     *  Set Serial IO type as "Legacy";
+     *  Set Base address of serial PIO region as "0x3f8";
+     *  Set 2M-aligned Start physical address of the RAM region used by the
+        hypervisor as "0x6a000000";
+     *  Select "Enable ACPI DMA Remapping tables parsing".
+
+   Other items is better to keep the default value.
+
+#. Check GRUB boot loader name. 
+
+   Double check GRUB loader name whether have "GRURB" prefix on ubuntu of target
+   board. If no, should update it:
```

Comment:
@leilu Can you remove this duplicate statement? "Check GRUB boot loader name" is enough. Thanks!

Reply:
fix at doc v3 version.